lib/portage/news.py

```diff
@@ -435,9 +427,7 @@ def isValid(self):
 
     def checkRestriction(self, **kwargs):
         vdb = kwargs["vardb"]
```

Comment:
I wonder if we should just inline `vdb` too and do:
`return kwargs["vardb"].match(self.atom)`
given e.g. `checkRestriction()` does something similar.

But this is a total nitpick and it's OK to not bother. I kind of think `vdb = ...` is _slightly_ more readable. Your call I think?

Reply:
I have thought of it too.

I just was conserving the existing style, so it lowers the chances of @mattst88 yelling at me for style changes.

Reply:
fwiw I don't think matt minds style changes, just not mixed in with functional ones (so it's easier to spot them and review them)